data/mods/Perk_melee/PERKS/insight.json

```diff
@@ -0,0 +1,71 @@
+[
+  {
+    "type": "mutation",
+    "id": "MELEE_PERK_INSIGHT",
+    "copy-from": "MELEE_PERK_BASE",
+    "name": { "str": "Insight I" },
+    "description": "You passively accumulate up to 20 insight stacks while near enemies.  Each insight stack increases all damage dealt by 1 until your next attack.  Insight greatly favors perceptive warriors.",
+    "flags": "MELEE_PERK_INSIGHT",
+    "enchantments": [ "melee_perk_ench_insight" ]
+  },
+  {
+    "type": "mutation",
+    "id": "MELEE_PERK_INSIGHT_DODGE",
+    "copy-from": "MELEE_PERK_BASE",
+    "name": { "str": "Like Water" },
+    "description": "You gain a +25 dodge chance if you have at least 10 insight stacks.",
+    "flags": "MELEE_PERK_LIKE_WATER",
```

Comment:
This flag seems to be unused?

Reply:
I keep all these flags for cross modability or for future items having the techniques incorporated even if you dont have the perks.

Most are unused atm,